libs/activities/pom.xml

```diff
@@ -36,11 +36,6 @@
                          was the maven release plugin was not committing during the release.
                      -->
                     <version>2.4.1</version>
```

Comment:
I need to test whether I can remove these old version overrides of the release plugin, but I _think_ they shouldn't be necessary now